packages/graph/migrations/postgres/migration/1656417312397_initial.ts

```diff
@@ -132,11 +132,13 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
         type: "UUID",
         notNull: true,
         references: "property_types",
+        onDelete: "CASCADE",
```

Comment:
@TimDiekmann you don't have to answer, I just wonder if anyone in @hashintel/backend has an idea

Reply:
We currently only have two options to properly run the tests: 
- Set up a database for each test, which has to be dropped after running the test
- Cleaning up the database after each test has run

We decided against the first solution for now for simplicity. The second solution requires a deletion method for inserted items. Originally, we don't want to expose the database, however, then we can't clean up the database for tests. Cascading is still easier than implementing a recursive deletion of all entries, as we only rely on two tables rather than on *all* tables.

Before we use the first solution, I don't see us removing the cascading.

Reply:
I think we should explicitly mention this then in the task about E2E tests. Because right now we are polluting our table schemas with dangerous unwanted functionality purely for the benefit of ergonomics of testing.

Reply:
Have added it.